website/docs/r/machine_learning_compute_cluster.html.markdown

```diff
@@ -0,0 +1,162 @@
+  vm_size                       = "Standard_DS2_v2"
+  machine_learning_workspace_id = azurerm_machine_learning_workspace.example.id
+  subnet_resource_id            = azurerm_subnet.example.id
+
+  scale_settings {
+    min_node_count                   = 0
+    max_node_count                   = 1
+    node_idle_time_before_scale_down = "PT30S" # 30 seconds
+  }
+
+  identity {
+    type = "SystemAssigned"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `identity` - (Required) A `identity` block as defined below. Changing this forces a new Machine Learning Compute Cluster to be created.
+
+* `location` - (Required) The Azure Region where the Machine Learning Compute Cluster should exist. Changing this forces a new Machine Learning Compute Cluster to be created.
+
+* `machine_learning_workspace_id` - (Required) The ID of the Machine Learning Workspace. Changing this forces a new Machine Learning Compute Cluster to be created.
+
+* `name` - (Required) The name which should be used for this Machine Learning Compute Cluster. Changing this forces a new Machine Learning Compute Cluster to be created.
+
+* `scale_settings` - (Required) A `scale_settings` block as defined below. Changing this forces a new Machine Learning Compute Cluster to be created.
+
+* `vm_priority` - (Required) The priority of the VM. Changing this forces a new Machine Learning Compute Cluster to be created.
+
+* `vm_size` - (Required) The size of the VM. Changing this forces a new Machine Learning Compute Cluster to be created.
+
+---
+
+* `description` - (Optional) The description of the Machine Learning compute. Changing this forces a new Machine Learning Compute Cluster to be created.
+
+* `subnet_resource_id` - (Optional) The ID of the Subnet that the Compute Cluster should reside in. Changing this forces a new Machine Learning Compute Cluster to be created.
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the Machine Learning Compute Cluster. Changing this forces a new Machine Learning Compute Cluster to be created.
+
+---
+
+A `identity` block supports the following:
+
+* `type` - (Required) The Type of Identity which should be used for this Disk Encryption Set. At this time the only possible value is SystemAssigned. Changing this forces a new Machine Learning Compute Cluster to be created.
+
+---
+
+A `scale_settings` block supports the following:
+
+* `max_node_count` - (Required) Maximum node count. Changing this forces a new Machine Learning Compute Cluster to be created.
+
+* `min_node_count` - (Required) Minimal node count. Changing this forces a new Machine Learning Compute Cluster to be created.
+
+* `node_idle_time_before_scale_down` - (Required) Node Idle Time Before Scale Down: defines the time until the compute is shutdown when it has gone into Idle state. Is defined according to W3C XML schema standard for duration. Changing this forces a new Machine Learning Compute Cluster to be created.
```

Comment:
would this make more sense as
```suggestion
* `scale_down_nodes_after_idle_duration` - (Required) Node Idle Time Before Scale Down: defines the time until the compute is shutdown when it has gone into Idle state. Is defined according to W3C XML schema standard for duration. Changing this forces a new Machine Learning Compute Cluster to be created.
```
?

Reply:
That is the azure cli term used here:  https://docs.azure.cn/zh-cn/cli/ext/azure-cli-ml/ml/computetarget/create?view=azure-cli-latest#ext_azure_cli_ml_az_ml_computetarget_create_amlcompute. But I agree that your proposal is more descriptive, so up to you.

Reply:
lets use this more description one, we tend to change names to better match the terraform schema/provider the best UX

Reply:
addressed in newest commit